docker/wc-admin-php-test-suite/docker-compose.yml

```diff
@@ -2,15 +2,19 @@ version: '3'
 
 services:
     phpunit:
-        build: '.'
-        image: wc-admin-php-test-suite-phpunit:1.2.1
+        build:
+            context: '.'
+            args:
+                - PHP_VERSION=${PHP_VERSION:-7}
+        image: wc-admin-php-${PHP_VERSION:-7}-test-suite-phpunit:1.2.2
```

Comment:
Added the php version to the image names and also the volume name, so that we have a seperate image for PHP versions. This way it won't need to rebuild every time when switching versions.